src/clients/python/experimental_api_v2/library/grpcclient.py

```diff
@@ -0,0 +1,674 @@
+        self._input.contents.CopyFrom(self._input_contents)
+
+    # FIXMEPV2: Add parameter support
+    @property
+    def parameters(self):
+        raise_error("Not implemented yet")
+
+    def _get_tensor(self):
+        """Retrieve the underlying InferInputTensor message.
+        Returns
+        -------
+        Protobuf Message 
+            The underlying InferInputTensor protobuf message.
+        """
+        return self._input
+
+
+class InferOutput:
+    """An object of InferOutput class is used to describe a
+    requested output tensor for an inference request.
+
+    Parameters
+    ----------
+    name : str
+        The name of output tensor to associate with this object
+        
+    data_format : str
+        The format to use when returning the ouput tensor data. Options
+        are "explicit", "binary" and "shared_memory".
+        Default is "binary". If "shared_memory" is specified then
+        the "shared_memory_data" field must also be specified.
+        Server support for “binary” and “shared_memory”
+        is optional.
+    """
+
+    def __init__(self, name, data_format="binary"):
+        self._output = grpc_service_v2_pb2.ModelInferRequest(
+        ).InferRequestedOutputTensor()
+        self._output.name = name
+        self._output.data_format = data_format
+
+    @property
+    def name(self):
+        """Get the name of output associated with this object.
+
+        Returns
+        -------
+        str
+            The name of output
+        """
+        return self._output.name
+
+    @property
+    def data_format(self):
+        """Get the requested format of ouput associated with this object.
+
+        Returns
+        -------
+        str
+            The data format in which output will be requested
+        """
+        return self._output.data_format
+
+    @data_format.setter
+    def data_format(self, data_format):
+        """Set the requested format of ouput associated with this object.
+
+        Parameter
+        ---------
+        data_format : str
+            The data format in which output will be requested.
+        """
+        self._output.data_format = data_format
+
+    # FIXMEPV2: Add parameter support
+    @property
+    def parameters(self):
+        raise_error("Not implemented yet")
+
+    def _get_tensor(self):
+        """Retrieve the underlying InferRequestedOutputTensor message.
+        Returns
+        -------
+        Protobuf Message 
+            The underlying InferRequestedOutputTensor protobuf message.
+        """
+        return self._output
+
+
+class InferResult:
+    """An object of InferResult class holds the response of
+    an inference request and provide methods to retrieve
+    inference results.
+
+    Parameters
+    ----------
+    result : Protobuf Message
+        The ModelInferResponse returned by the server
+    """
+
+    def __init__(self, result):
+        self._result = result
+
+    def get_output_in_numpy(self, name):
```

Comment:
How about just "as_numpy" since this class is already known to be an output/result.